server/server.go

```diff
@@ -2365,3 +2375,15 @@ func parseURL(s string, useHTTPSByDefault bool) (*url.URL, error) {
 	}
 	return url.Parse(s)
 }
+
+func getProvenance(revision string) map[string]interface{} {
+	p := make(map[string]interface{})
```

Comment:
Instead of making a new one of these on each API call can we create & initialize it in the Server's Init() and stash it on the server to re-use in responses?

Means switching to like:
```
if provenance {
	result.Provenance = s.provenence
}
```

Reply:
For now, let's not change this. If we want to cache the provenance on the server we can but we'll need to make sure it updates when bundles active, etc. We could do this in the server's `reload` function but it's more work than needed to make this land. Once we have it in place, we can refactor the implementation if needed.

Reply:
Ohhh. I misunderstood what `s.revision` was :D I was thinking it didn't change through the servers lifespan. Totally agree that we shouldn't mess around with it now.